docs/docs/codes/modifiers.mdx

```diff
@@ -30,13 +30,17 @@ These act like any other key code.
 
 ### Modifier Functions
 
+Modifier functions add one or more modifiers to a keycode.
```

Comment:
~~key~~code
? - to match with everything else.

Incidentally that's also why I had that word italicized originally.